app/views/candidate_interface/gcse/type/edit.html.erb

```diff
@@ -10,7 +10,7 @@
 
       <%= f.govuk_error_summary %>
 
-      <%= f.govuk_radio_buttons_fieldset :qualification_type, legend: { size: 'm', text: t('application_form.gcse.qualification_type.label') } do %>
+      <%= f.govuk_radio_buttons_fieldset :qualification_type, legend: { size: 'm', text: t('application_form.gcse.qualification_type.label'), tag: 'span' } do %>
```

Comment:
Any reason not to use h2s or h3s?

Reply:
Surely `<legend>` provides the required structure and semantics?